log-manager/src/main/java/io/airlift/log/Logging.java

```diff
@@ -181,7 +175,7 @@ public void setLevel(String loggerName, Level level)
         return levels.build();
     }
 
-    private void processLevels(Properties properties)
+    private void processLevels(Map<Object, Object> properties)
```

Comment:
Shall we expect key/values to be Strings rather than being unnecessarily flexible with .toString()?

Reply:
Good idea. Switched the caller to use `Maps.fromProperties()`